app/test_main.py

```diff
@@ -1,3 +1,51 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (100, 100, [21, 17])
+    ],
+    ids=[
+        "check if cat and dog age 0 equals 0",
+        "cat and dog age 14 should be equal to 0",
+        "cat and dog age 15 should be equal to 1",
+        "cat and dog age 23 should be equal to 1",
+        "cat and dog age 24 should be equal to 3",
```

Comment:
There is a mismatch between the expected value in the parameterized test and the description in the `ids` list. The description states that the expected value should be 3, but the test case expects 2. Please ensure consistency between the test case and its description.